spec.html

```diff
@@ -28375,7 +28375,7 @@ <h1>Math.fround ( _x_ )</h1>
 
       <emu-clause id="sec-math.hypot">
         <h1>Math.hypot ( _value1_, _value2_, ..._values_ )</h1>
-        <p>`Math.hypot` returns an implementation-approximated value representing the square root of the sum of squares of its arguments.</p>
+        <p>`Math.hypot` returns the square root of the sum of squares of its arguments.</p>
```

Comment:
Nit: this summary stands out in that it mentions the method name again at the beginning. The others just start with "Returns ...".

Reply:
Fixed!